server/src/main/java/io/crate/planner/operators/InsertFromValues.java

```diff
@@ -486,25 +504,27 @@ public List<CompletableFuture<Long>> executeBulk(DependencyCarrier dependencies,
         );
     }
 
-    private void checkConstraintsOnGeneratedSource(@Nullable IndexItem indexItem,
-                                                   String indexName,
-                                                   DocTableInfo tableInfo,
-                                                   PlannerContext plannerContext,
-                                                   HashMap<String, Consumer<IndexItem>> validatorsCache) throws Throwable {
-        if (indexItem == null) {
-            return;
-        }
-        var validator = validatorsCache.computeIfAbsent(
-            indexName,
-            index -> Indexer.createConstraintCheck(
+    public static BiConsumer<String, IndexItem> createConstraintsCheck(DocTableInfo tableInfo,
```

Comment:
Why not leave it as a `void` return and just create a lambda on usage? 
```suggestion
    public static void checkConstraints(DocTableInfo tableInfo,
```

Reply:
Thx, applied it and also adjusted slightly in https://github.com/crate/crate/pull/14373/commits/bc24962547a73b116e8ffe8613d987d712e0736e